applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application_variables.cpp

```diff
@@ -11,12 +11,20 @@ KRATOS_CREATE_VARIABLE(double, ADJOINT_VELOCITY_POTENTIAL)
 KRATOS_CREATE_VARIABLE(double, ADJOINT_AUXILIARY_VELOCITY_POTENTIAL)
 
 // Flow field magnitudes
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_INFINITY)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_LOWER)
 KRATOS_CREATE_VARIABLE(double, PRESSURE_LOWER)
 KRATOS_CREATE_VARIABLE(double, POTENTIAL_JUMP)
 KRATOS_CREATE_VARIABLE(double, ENERGY_NORM_REFERENCE)
 KRATOS_CREATE_VARIABLE(double, POTENTIAL_ENERGY_REFERENCE)
+KRATOS_CREATE_VARIABLE(double, CP)
+KRATOS_CREATE_VARIABLE(double, GAMMA)
+
+// Free stream magnitudes
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_INFINITY)
+KRATOS_CREATE_VARIABLE(double, DENSITY_INFINITY)
+KRATOS_CREATE_VARIABLE(double, MACH_INFINITY)
+KRATOS_CREATE_VARIABLE(double, PRESSURE_INFINITY)
+KRATOS_CREATE_VARIABLE(double, AOA)
```

Comment:
AOA stands for?

Reply:
AOA stands for ANGLE_OF_ATTACK, I will change it.